src/Core/NotificationsManager.php

```diff
@@ -344,6 +344,10 @@ private function formatNotifs(array $notifs, $ident = "")
 						break;
 
 					case ACTIVITY_FRIEND:
+						if (!isset($it['object'])) {
```

Comment:
Could it be related to the fact that the base item table doesn't use `contact` anymore?

Reply:
We will see what's the root of the problem is.